src/update.fs

```diff
@@ -0,0 +1,186 @@
+
+module UpdateCmd
```

Comment:
it's the 1-1 conversion from `src/update.cmd`. 
Yuo are right, i'll add a comment with the original .bat file path in the .fs files, it's easier to understand.